contracts/common/lib/DepositsTracker.sol

```diff
@@ -134,65 +115,91 @@ library DepositsTracker {
                 ++i;
```

Comment:
since solidity v 0.8.0 there is no need to use `unchecked` block in `for` loops as it handled automatically by the compiler , just use as usual: `for(uint i; i<N; ++i) ...`